src/networks.json

```diff
@@ -737,6 +737,23 @@
     "start": 15123138,
     "imageIPFS": "QmYqkn8pJUaV9KcEPYEvRPwgbfeozLEvcQ9aEwKNRUL3cR"
   },
+  "941": {
+    "key": "941",
+    "name": "PulseChain Testnet V2B",
+    "shortName": "pulsechain",
+    "chainId": 941,
+    "network": "Testnet v2B",
+    "multicall": "0x959a437F1444DaDaC8aF997E71EAF0479c810267",
```

Comment:
HI @aggris2 Can you verify and publish code here https://scan.v2b.testnet.pulsechain.com/address/0x959a437F1444DaDaC8aF997E71EAF0479c810267/contracts

Reply:
@ChaituVR done!